stable/sumologic-fluentd/Chart.yaml

```diff
@@ -1,7 +1,8 @@
 apiVersion: v1
 name: sumologic-fluentd
-version: 1.1.1
+version: 2.0.0
 appVersion: 2.3.0
+kubeVersion: "^1.10.0-0"
```

Comment:
```suggestion
kubeVersion: ">= 1.10"
```

🙂

Reply:
Otherwise LGTM

Reply:
ok

Reply:
fyi: it caused pull-charts-e2e fail
restored